pjsip/src/pjsip-ua/sip_inv.c

```diff
@@ -2061,6 +2061,15 @@ static pj_status_t inv_check_sdp_in_incoming_msg( pjsip_inv_session *inv,
 	{
 	    const pjmedia_sdp_session *reoffer_sdp = NULL;
 
+	    if (pjmedia_sdp_neg_get_state(inv->neg) !=
+	    	PJMEDIA_SDP_NEG_STATE_DONE)
+	    {
+	    	PJ_LOG(4,(inv->obj_name, "Ignoring %s response "
+		          "because SDP negotiation is in progress",
+		          (st_code/10==18? "early" : "final" )));
+	    	return PJ_EINVALIDOP;
```

Comment:
- shouldn't it be `Ignoring message body in %s response` instead of `Ignoring %s response`?
- in the `else` block (which also ignoring message body), it returns `PJ_SUCCESS`, any particular reason it is not the case here?